core/src/unprocessed_packet_batches.rs

```diff
@@ -372,9 +376,20 @@ pub fn packet_message(packet: &Packet) -> Result<&[u8], DeserializedPacketError>
         .ok_or(DeserializedPacketError::SignatureOverflowed(sig_size))
 }
 
-/// Computes `(addition_fee + base_fee / requested_cu)` for `deserialized_packet`
-fn compute_fee_per_cu(_message: &SanitizedVersionedMessage, _bank: &Bank) -> u64 {
-    1
+fn compute_fee_per_cu(message: &SanitizedVersionedMessage) -> Option<FeePerCu> {
```

Comment:
maybe name this `prioritization_fee_per_cu`? Otherwise you would think it represents the full fee of a tx (including sig fees)

Reply:
makes sense, changed